frontend/src/report/report-page/encoding/Encoding.svelte

```diff
@@ -88,12 +196,27 @@
 						}
 					}} />
 			</div>
-			<svelte:component this={EncodingMap[parameters.xEncoding]} />
+			{#if chartType === ChartType.BEESWARM}
+				<label>
+					<input
+						type="radio"
+						bind:group={$reports[$report].parameters.fixedDimension}
+						name="fixed_dimension"
+						value={"x"} />
+					<span> Fix this dimension (Dropdown)</span>
```

Comment:
Maybe remove `(Dropdown)`? I don't really understand it

Reply:
Instead, can we add a tooltip (like the ones next to `slices` and `metadata` in explore page) that explains it a little bit?

Reply:
Sure, will add the tooltip to make it more understandable and remove the (dropdown), it's just a mistake LOL